gatk_base_recalibrator_4.1.8.1/gatk_base_recalibrator_4.1.8.1.cwl

```diff
@@ -0,0 +1,349 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  dct: 'http://purl.org/dc/terms/'
+  doap: 'http://usefulinc.com/ns/doap#'
+  foaf: 'http://xmlns.com/foaf/0.1/'
+  sbg: 'https://www.sevenbridges.com/'
+id: gatk_base_recalibrator_4_1_8_1
+baseCommand:
+  - gatk
+inputs:
+  - id: input
+    type: File
+    inputBinding:
+      position: 3
+      prefix: '--input'
+    doc: BAM/SAM file containing reads
+    secondaryFiles:
+      - ^.bai
+  - id: known_sites_1
```

Comment:
I am no expert in cwl, but wouldn't it be better if we allow the user to specify this option more than two times (instead of just `known_sites_1` and `known_sites_2`)? Not sure if we need to specify known sites more than two times for the pipeline, but I think that would be more generalizable that way.

Reply:
Thanks for catching all the issues. fixed this as well.